presto-main/src/main/java/com/facebook/presto/operator/PartitionedOutputOperator.java

```diff
@@ -298,14 +310,17 @@ public Page getOutput()
         private final List<Optional<Block>> partitionConstants;
         private final PagesSerde serde;
         private final List<PageBuilder> pageBuilders;
+        private final boolean replicateAny;
```

Comment:
sure, but `replicateFirstRow` doesn't add ambiguity to the code and is effectively what we're doing. I'm OK with replacing the `firstRow` part with sth less concrete - as long as it's not introducing ambiguity